pantahub.c

```diff
@@ -152,8 +152,14 @@ const char **pv_ph_get_certs(struct pantavisor *__unused)
 
 	pv_paths_cert(path, PATH_MAX, "");
 	n = scandir(path, &files, NULL, alphasort);
-	if (n < 0)
+	if (n < 0) {
+		pv_log(WARN, "%s could not be scanned", path);
```

Comment:
imo certs that cannot be parsed should trigger a rollback also? feels like something bogus was shipped...

Reply:
We don't disable anything. We just try to make the requests but we get no response from ph in case there are no certs or there are bad certs.

About the rollback thing... if the requests are not successful, it should rollback eventually because of no response from ph.

Reply:
Maybe we just should avoid making the request altogether and let the rest flow with the rollback etc